### PR TITLE
Add JS SDK & Livestream support

### PR DESCRIPTION
**Pull request by mlejva**
This PR adds JS SDK to for the Desktop Sandbox.

Todo:
- [x] Parity with Python SDK
- [x] Docstrings
- [x] Livestream support in JS SDK
- [x] Livestream support in Python SDK
- [ ] Tests (Will do in a separate PR)
   - [ ] Node tests
   - [ ] Bun tests
   - [ ] Deno tests
- [x] Publish SDKs
   - [x] Prerequisite: merge https://github.com/e2b-dev/E2B/pull/478

Cleanup:
- [ ] Updated readme with JS snippets
- [x] Move Python SDK to `packages/`

### CODE REVIEW

**Review by ValentaTomas — APPROVED** [review]

File: python/e2b_desktop/main.py [hunk 1]
```diff
@@ -1,37 +1,120 @@
...
 from e2b import Sandbox as SandboxBase
+import requests
 
 
 class Sandbox(SandboxBase):
     default_template = "desktop"
+    stream_base_url = "https://e2b.dev"
+
+    @staticmethod
+    def start_video_stream(sandbox, api_key, sandbox_id):
```
Comment (ValentaTomas) [inline python/e2b_desktop/main.py:18]: Please add types for the parameters.

File: pnpm-workspace.yaml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 packages:
-  - js
+  - packages/js-sdk
   - python
```
Comment (ValentaTomas) [inline pnpm-workspace.yaml:3]: Why is the JS package in `./packages/js-sdk` and the Python in `./python`?

Comment (mlejva) [reply pnpm-workspace.yaml:3]: Because this PR isn't done yet, it's a draft

Comment (mlejva) [reply pnpm-workspace.yaml]: Both in `packages` now

**Pull request merged**